Source/Classes/OverlayContainerViewController.swift

```diff
@@ -325,22 +294,21 @@ open class OverlayContainerViewController: UIViewController {
     }
 
     private func loadOverlayViews() {
-        guard !viewControllers.isEmpty else { return }
-        groundView.isHidden = viewControllers.count == 1
+        widthLimitConstraint?.isActive = false
+        widthLimitConstraint = nil
+        groundView.isHidden = viewControllers.count <= 1
         var truncatedViewControllers = viewControllers
         truncatedViewControllers.popLast().flatMap {
             addChild($0, in: overlayContainerView, pinToContainer: false)
             $0.view.translatesAutoresizingMaskIntoConstraints = false
             $0.view.topAnchor.constraint(equalTo: overlayContainerView.topAnchor).isActive = true
             $0.view.bottomAnchor.constraint(equalTo: overlayContainerView.bottomAnchor).isActive = true
             $0.view.centerXAnchor.constraint(equalTo: overlayContainerView.centerXAnchor).isActive = true
-            overlayContainerLandscapeWidthConstraint = $0.view.widthAnchor.constraint(equalToConstant: 375)
-            overlayContainerPortraitWidthConstraint = $0.view.widthAnchor.constraint(equalTo: overlayContainerView.widthAnchor)
```

Comment:
The call to `addChild` and the creation of these constraints was being repeated with each orientation change, resulting in an ever-increasing number of duplicate constraints. This was not in itself a cause of invalid constraints but could certainly tax the autolayout system after many rotations.